code/modules/mob/living/simple_animal/friendly/drone/_drone.dm

```diff
@@ -3,8 +3,10 @@
 #define DRONE_HEAD_LAYER 2
 #define DRONE_TOTAL_LAYERS 2
 
-#define DRONE_NET_CONNECT "<span class='notice'>DRONE NETWORK: [name] connected.</span>"
-#define DRONE_NET_DISCONNECT "<span class='danger'>DRONE NETWORK: [name] is not responding.</span>"
+#define get_area(A) (isarea(A) ? A : get_step(A, 0)?.loc)
```

Comment:
```suggestion
```
`#define get_area(A) (isarea(A) ? A : get_step(A, 0)?.loc)` is already defined identically in yogstation13/Yogstation/blob/master/code/__DEFINES/misc.dm, line 184.

There is no reason to define it a second time.

Reply:
Doing this will make Linter happy as well

Reply:
updated :)